app/static/openapi.yaml

```diff
@@ -318,11 +313,6 @@ paths:
                     name: 'Android'
                   - id: 20
                     name: 'CSS'
-                has_next: true
-                has_prev: false
-                number_of_pages: 4
-                page: 1
-                records_per_page: 20
                 status: 'ok'
                 status_code: 200
                 total_count: 73
```

Comment:
This total count should not be 73 unless we add 73 languages to the above list.. I'm thinking we make it match what's in the local development DB right now, which is 23, and make the list match the 23 langauges we have in there.

Reply:
I just noticed we have Linux listed twice. Should I include it anyway or leave it off the list and make the total count 22?

```json
{
  "id": 3, 
  "name": "Unix/Linux/*nix"
}
```

```json
{
  "id": 23, 
  "name": "Linux"
}
```

Reply:
It's because we have it twice in the resources.yml

https://github.com/OperationCode/resources_api/blob/main/resources.yml#L87

https://github.com/OperationCode/resources_api/blob/main/resources.yml#L1263

Would be useful to change that if we change this. So it's up to how motivated you are to change it all I guess

Reply:
It seems appropriate to change all the `"Linux"` resources to `"Unix/Linux/*nix"`. Changing _The Art of Unix Programming_ to `"Linux"` wouldn't really be accurate. I can add this to my next commit.

Reply:
Do you want to change it in prod too? That would be helpful I would think.